src/SourceBuild/tarball/content/repos/runtime-portable.proj

```diff
@@ -1,19 +1,11 @@
 <Project>
-
-  <PropertyGroup>
-    <SourceDirectory>runtime</SourceDirectory>
-    <PortableBuild>true</PortableBuild>
```

Comment:
Does this still build portable without this?  I'm guessing we just stopped fighting the runtime default and the non-portable runtime build has to be special instead?

Reply:
It does build portable.  When runtime non-portable gets added, I'll need to figure out what the "special" is.